emark/contrib/dashboard/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.admin.views.decorators import staff_member_required
+from django.urls import include, path
+
+from . import views
+
+app_name = "emark-dashboard"
+
+urlpatterns = [
+    path("", staff_member_required(views.DashboardView.as_view()), name="dashboard"),
+    path(
+        "<str:email_class>/",
```

Comment:
I would feel better if email_class was a slug. The urls look quite unusual right now. Perhaps you can do some magic to "transform" it back and forth.

Reply:
I initially felt the same. But it's not too uncommon to do so in technical documentations. Python.org does it too. Furthermore, none of the characters are illegal.
Slugs would only make sense for SEO, which we don't have to care about. All in all, it would add more complexity. Therefore, I decided not to do it.

Reply:
> I initially felt the same. But it's not too uncommon to do so in technical documentations. Python.org does it too. Furthermore, none of the characters are illegal.

That was my line of thought as well.